src/OpenLoco/src/Ui.cpp

```diff
@@ -113,9 +112,10 @@ namespace OpenLoco::Ui
         return _screenInfo->height;
     }
 
+    // TODO: Rename misleading name.
```

Comment:
should we rename this here?

Reply:
Best done in a separate PR tbh, I want to clean up a few things after this PR.